README.md

```diff
@@ -1,2 +1,63 @@
 # pyweek36
 PyWeek 36
+
+
+
+# Mechanics
+
+Side-scrolling 2D platform game over a grid system. Each grid cell contains one of three types of blocks:
+
+1. Air; the player can walk/fall through.
+2. Solid Block; the player collides with and thus can walk over, but not through.
+3. Dark Matter Block; usually behaves as Air.
+
+To win, the player has to reach the end of the map, either the top or the extreme right side (without dying).
+
+Each level is composed of a map built with the above 3 block types and a set of the following additional mechanics.
+
+## Additional Mechanics
+
+These mechanics are to be explored, with the following priority. Some might fall off or change slightly as we progress.
+
+1. Dark Matter spreads to adjacent Solid Blocks, both opening and closing pathways as the time passes. It can also remove the floor from under the player's feet, and falling to the bottom of the map equals death.
+2. The player has a point and shoot device which turns Dark Matter blocks into Solid-like blocks.
+    1. Turns Dark Matter blocks into Solid Blocks.
+    2. Turns Dark Matter blocks into Solidified Dark Matter, which behaves like Solid Blocks but goes back into being regular Dark Matter after a time.
+    3. A fourth block type, Dark Matter Source, is added as a Dark Matter block immune to conversion.
+        * Can facilitate the design of levels where the player is not able to erradicate the Dark Matter.
+    4. Can also turn Solid Blocks into Dark Matter (might [be incoherent](https://github.com/letsbuilda/pyweek36/pull/12/files#r1328208664)).
+3. The player has a device which allows him to interact with Dark Matter blocks as if they were Solid Blocks, effectively toggling the behavior of all Dark Matter blocks at the map when turned on.
+    1. The device overheats after a fixed time being used, making it go into cooldown. The player has to control how much it uses it.
+    2. The device flickers periodically, making it so that the player, given a short moment of warning, has to run to a safe Solid Block.
+
```

Comment:
an idea of mine

since dark matter is made into something that spreads, what if we make it something that the player is running away from? if we step on it, it drains health a set amount (and maybe the longer we step on it the health lost becomes exponentially larger).

we choose a default, volatile block as our 'base block' (like the `air block` that maybe disappear after a set amount of time after the player stepped on it) and the gun is used to convert `base blocks` into whatever we want for flexibility. 

Say, the presence of dark matter on the level introduces `hostile blocks`, which can be neutralized or made less hostile with the gun (player will still lose health, but the choice of which block the player neutralize will affect how much health percentage they ended up with) and we introduce health pick-ups to replenish it.

Reply:
I think we all agree on the player running away from the spreading dark matter. I think of the gun device as essentially a way to break solid blocks to clear a path for the player. Maybe there could be different guns as well. That being said I think this discussion should continue when we've made specific issues for the device.